xchange-bitfinex/src/main/java/info/bitrich/xchangestream/bitfinex/BitfinexStreamingExchange.java

```diff
@@ -15,15 +15,17 @@
 
     private final BitfinexStreamingService streamingService;
     private BitfinexStreamingMarketDataService streamingMarketDataService;
+    private BitfinexStreamingRawService streamingAuthenticatedDataService;
 
     public BitfinexStreamingExchange() {
         this.streamingService = new BitfinexStreamingService(API_URI);
+        this.streamingAuthenticatedDataService = new BitfinexStreamingRawService(API_URI);
```

Comment:
Do we need to have a second connection to BFX for the authenticated stream, or is it possible to just authenticate the existing connection?

Reply:
According to the BFX documentation, it very much looks like this is the case, so I'm going to try it.